{"name": "test_invalid_login[chrome]", "status": "passed", "attachments": [{"name": "log", "source": "490c20f5-b1a6-4ec9-bc56-0a855b831f43-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "setup_driver", "value": "'chrome'"}], "start": 1682650769471, "stop": 1682650782515, "uuid": "55c41a82-c142-4cb6-908c-d6707f66a7a1", "historyId": "8c9b43ce6495e3ebb09003d420ea79e1", "testCaseId": "0d4e817f6e9d6168955d79b4f04c8a6a", "fullName": "test_invalidLogin.Test_006_Login#test_invalid_login", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_driver')"}, {"name": "suite", "value": "test_invalidLogin"}, {"name": "subSuite", "value": "Test_006_Login"}, {"name": "host", "value": "DESKTOP-48C1KTV"}, {"name": "thread", "value": "23860-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_invalidLogin"}]}